__test__/integration/loan_offer/runners/create_and_sign_as_creditor.ts

```diff
@@ -2,12 +2,14 @@ import { Dharma } from "../../../../src";
 
 import { LoanOffer } from "../../../../src/types";
 
-export async function testCreateAndSignAsCreditor(dharma: Dharma, params: any) {
+import { DebtOrderParams } from "../../../../src/loan/debt_order";
+
+export async function testCreateAndSignAsCreditor(dharma: Dharma, params: DebtOrderParams) {
     describe("passing valid params", () => {
         let loanOffer: LoanOffer;
 
         beforeAll(async () => {
-            loanOffer = await LoanOffer.createAndSignAsCreditor(dharma, params);
+            loanOffer = await LoanOffer.createAndSignAsCreditor<LoanOffer>(dharma, params);
```

Comment:
don't believe you need to specify type here because `createAndSignAsCreditor` only lives on `LoanOffer`

Reply:
done